samples/bluetooth/nrf_dm/README.rst

```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an offset.
+The offset value may be due to the design of the radio circuit, the antenna used or the designed PCB Layout.
+Zero-distance offset should be compensated for by the user of the library and sample.
+Users of the library need to manually perform the calibration.
```

Comment:
```suggestion
```
Okay to combine this with what was said below, as I'm suggesting? Does it change the meaning?

Reply:
I think this sentence is not needed.